utils/user_info.py

```diff
@@ -38,7 +39,7 @@ def has_c_ownership(curr_user, curr_contest):
     except Contest.DoesNotExist:
         logger.warning('Contest id %ld does not exsit!' % curr_contest.id)
 
-    ownership = (curr_user.username == curr_contest.owner)
+    ownership = (curr_user == curr_contest.owner)
```

Comment:
Two points want to check:
(1) In my opinion, ownership -> isOwner is better.
(2) Is ```curr_user``` an user object? (I think previous design is more elegant for the future)